src/main/java/com/google/api/codegen/transformer/StaticLangApiMethodTransformer.java

```diff
@@ -690,8 +690,6 @@ private void setCallableMethodFields(
       callableImplType = ApiCallableImplType.of(methodConfig.getGrpcStreamingType());
     } else if (methodConfig.isBatching()) {
       callableImplType = ApiCallableImplType.BatchingApiCallable;
-    } else if (methodConfig.isLongRunningOperation()) {
-      callableImplType = ApiCallableImplType.InitialOperationApiCallable;
```

Comment:
Why is this `if {}` block being removed?

Reply:
Because now we don't need to distinguish SimpleApiCallable and InitialOperationApiCallable; the callable for initiating operations will now just be a SimpleApiCallable.